Committees/NPEC.adoc

```diff
@@ -0,0 +1,138 @@
+= National Political Education Committee
+
+== Purpose
+
+Political education of PLTCom members is the sole mission of this committee. It will provide all members a course in capitalism first and scientific socialism second followed by a course on party work. It will also maintain a comprehensive digital library of marxist political material as well as a physical library as possible. 
+
+== Structure
+- Committee Chair
+- Librarian
+- Teacher(s)
+
+=== Courses

```

Comment:
I think the courses should be in a separate file. In my head, this file is to outline the duties of the committee as a reference to people who want to join a committee. We absolutely should construct a course catalog. I have a draft syllabus in a different repo we could easily copy and paste over for this repo.

Reply:
Ah ok yeah that would be fine to move it there.